lib/fog/libvirt/requests/compute/list_networks.rb

```diff
@@ -25,12 +25,14 @@ def get_network_by_filter(filter)
           end
         end
 
+        # bridge name may not be defined in some networks, we should skip that in such case
         def network_to_attributes(net)
           return if net.nil?
+          bridge_name = net.bridge_name rescue ''
```

Comment:
Does `bridge_name` returns an exception?

Reply:
Could we rescue only the exception raised by it instead of silently swallow any error?

Reply:
it does, sadly it's quite generic - `Libvirt::Error` but you're right, it's better to specify it explicitly